lib/swaggerValidator/trafficValidator.ts

```diff
@@ -25,6 +26,8 @@ export class TrafficValidator {
   private trafficFiles: string[] = [];
   private specPath: string;
   private trafficPath: string;
+  private trafficOperation: Map<string, string[]> = new Map<string, string[]>();
+  public coverageResult: Map<string, number> = new Map<string, number>();
```

Comment:
As discussed, let's return array of JSON like [{"spec":"", "coveredOperations": number, "totalOperations": number, "coverageRate": number }]